src/relayFeeCalculator/chain-queries/ethereum.ts

```diff
@@ -70,15 +75,20 @@ export const SymbolMapping: { [symbol: string]: { address: string; decimals: num
 export const defaultAverageGas = 116006;
```

Comment:
Can this be removed now?

Reply:
Yep! I left it in as an artifact for the event that the frontend-v2 / relayer-v2 / calling application referenced the `defaultAverageGas` variable. However, it has no functional use in the code anymore. (Updated: [here](https://github.com/across-protocol/sdk-v2/pull/64/commits/ffa6704b82773bcbeedf8e5e525d188dd11b2be5))